### PR TITLE
Add clone option to docker controller

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@miltontony ready for review cc. @JayH5

File: requirements-dev.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ pytest-xdist
 pytest-flake8
```
Comment (JayH5) [inline requirements-dev.txt]: Should be able to remove `pytest-flake8` now

File: .travis.yml [hunk 2]
```diff
@@ -36,6 +36,7 @@ install:
 
 script:
   - if [[ -z "$NO_COVERAGE" ]]; then COVERAGE_OPT="--cov"; else COVERAGE_OPT=""; fi
+  - flake8 .
```
Comment (JayH5) [inline .travis.yml]: In a few projects we've been running `flake8` _after_ `py.test` as failures during tests are a bit more interesting than lint failures so we'd prefer if flake8 didn't block the test run. But it's up to you :)

Comment (saeedmb8) [inline .travis.yml]: I agree, will move it after `py.test`

**Comment by saeedmb8** [comment]
clone

**Comment by JayH5** [comment]
:+1: for the flake8/pytest stuff.. I donno about Django tho 😅

**Comment by miltontony** [comment]
:+1:

**Pull request merged**